### PR TITLE
KANBAN-173 First pass on Xenbase changes

### PR DESCRIPTION
**Pull request by christabone**
First pass at changes required to load Xenbase data.

Most of the work was done in the DIOPT pre-process pipeline (merging orthology from DIOPT + Xenbase) which will be pushed in the agr_diopt repo when ready (the files are finished and uploaded, still organizing the code a bit better).

I'm sure there will be additional changes to the loader before we're at 100%, since this is the first new MOD in >5 years. 

I'll also be adding more tests starting tomorrow, hopefully to be finished next week.

### CODE REVIEW

File: src/default_env_vars.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ REDOWNLOAD_FROM_FMS: False
 USING_PICKLE: False
 DEBUG: False
 GENERATE_REPORTS: False
-ALLIANCE_RELEASE: "0.0.0"
+ALLIANCE_RELEASE: "5.3.0"
```
Comment (oblodgett) [inline src/default_env_vars.yml:5]: These vars need to not be set and let them be sent in via ENV variables.

File: Makefile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 REG=100225593120.dkr.ecr.us-east-1.amazonaws.com
-ALLIANCE_RELEASE=5.2.0
+ALLIANCE_RELEASE=5.3.0
```
Comment (oblodgett) [inline Makefile:2]: These need to come in via ENV variables and not hardcoded.

File: src/config/default.yml [hunk 3]
```diff
@@ -1,20 +1,20 @@
-schemaVersion: 1.0.1.5
-releaseVersion: 5.2.0
+schemaVersion: 1.0.1.6
```
Comment (sierra-moxon) [inline src/config/default.yml:1]: I think I saw 1.0.1.7 go thru today - is that for later? :)

File: src/default_env_vars.yml (hunk 1 above)
Comment (christabone) [reply src/default_env_vars.yml:5]: Aren't they overwritten? Those are just defaults if they're not set by the environment (which they are in the GoCD pipeline).

**Review by sierra-moxon — APPROVED** [review]

File: Makefile (hunk 2 above)
Comment (christabone) [reply Makefile:2]: This is overwritten by the ENV variable.

File: src/config/default.yml (hunk 3 above)
Comment (christabone) [reply src/config/default.yml:1]: Ha! Yes, it should be 1.0.1.7.

File: src/default_env_vars.yml (hunk 1 above)
Comment (christabone) [reply src/default_env_vars.yml]: I just set the it to 0.0.0 to be safe.

File: Makefile (hunk 2 above)
Comment (christabone) [reply Makefile]: I just set the it to 0.0.0 to be safe.

File: .env [hunk 4]
```diff
@@ -1,4 +1,4 @@
 REG=100225593120.dkr.ecr.us-east-1.amazonaws.com
-ALLIANCE_RELEASE
+ALLIANCE_RELEASE=5.3.0
```
Comment (oblodgett) [inline .env:2]: @christabone @sierra-moxon is this file even used?

Comment (christabone) [reply .env:2]: I'm not sure, it looks like @mluypaert added it in Jan 2021?

**Comment by christabone** [comment]
I'm more than happy to consolidate all the ENV stuff @oblodgett, just trying to focus on the Xenbase at the moment so that @markquintontulloch and @valearna have data to work with starting tomorrow / next week.

I can do a general code cleanup early next week and fix that stuff.

File: .env (hunk 4 above)
Comment (mluypaert) [reply .env:2]: @christabone @oblodgett Yes, this file is being used by docker compose, so when running the loader locally, though I wouldn't hard-code a release number in this file, this variable simply needs to be declared here to enable passing a value through the Makefile (`make run`, `make run_test` etc).

File: Makefile (hunk 2 above)
Comment (mluypaert) [reply Makefile]: Is the Makefile used outside of local usage? If not, what's the harm in having the latest release defined in the Makefile? As @christabone indicated, the value of this value can always be overwritten through ENV variables, and it simplifies the use of the use of the Makefile (as in, you don't always need to go in and manually change that variable for it to work).

File: src/config/default.yml [hunk 5]
```diff
@@ -50,8 +50,12 @@ ONTOLOGY:
   - BTO
   - CHEBI
   - ZECO
+  - XBED
+  - XSMO
+  - XPO
+  - XAO
 ExpressionAtlas: [FB, SGD, WB, ZFIN, RGD, MGI, HUMAN]
-ORTHO: [FB, SGD, WB, ZFIN, RGD, MGI, HUMAN]
+ORTHO: [FB, SGD, WB, ZFIN, RGD, MGI, Xenbase, HUMAN]
```
Comment (oblodgett) [inline src/config/default.yml:58]: This I think should be XB

**Comment by christabone** [comment]
@oblodgett Should we merge?

**Pull request merged**